src/main/java/Deadline.java

```diff
@@ -0,0 +1,15 @@
+public class Deadline extends Task {
+
+    protected String by;
+
+    public Deadline(String description, String by) {
+        super(description);
+        this.by = by;
+    }
+
+    @Override
+    public String toString() {
+        return "[D]" + super.toString() + " (by: " + by + ")";
+    }
+
+}
```

Comment:
Excellent practice on having consistent coding standards across the different class files